dbConnectivityChecker.js

```diff
@@ -0,0 +1,44 @@
+const MongoClient = require('mongodb').MongoClient;
+
+class DbConnectivityChecker {
+    constructor(io) {
+        this.url = process.env.DATABASE || 'mongodb://0.0.0.0:27017';
```

Comment:
Please change the url from "0.0.0.0" to "localhost". The 0 one will not be supported on the node 14 on production.